### ISSUE TITLE
Overwrite NaN field not working

### ISSUE THREAD

**Issue by boomfly** [jagi/meteor-astronomy#704]
Suppose we have document:
```coffeescript
import { Class } from 'meteor/jagi:astronomy'

Docs = new Mongo.Collection 'docs'

Doc = Class.create
  name: 'Doc'
  collection: Docs
  fields:
    someNumber: Number
newDoc = new Doc {
  someNumber: NaN
}
newDoc.save()

```
And we try to change `someNumber` field with not `NaN` value it does not changing:
```coffeescript
doc = Doc.findOne newDocId
doc.someNumber = 10
doc.save()

doc = Doc.findOne newDocId
console.log doc.someNumber # Value still is NaN
```

**Comment by lukejagodzinski** [comment]
Can you create reproduction repository?

**Comment by boomfly** [comment]
@lukejagodzinski sure! https://github.com/boomfly/astronomy-nan-issue/blob/master/server/main.coffee

**Comment by lukejagodzinski** [comment]
@boomfly indeed there is a bug. It just uses `$inc` modifier for numbers instead of `$set`. It will be easy fix

**Comment by lukejagodzinski** [comment]
@boomfly fixed in v2.7.2

**Comment by boomfly** [comment]
@lukejagodzinski thanks!
